resharper/resharper-unity/src/CSharp/Daemon/Stages/Analysis/MultiplicationOrderAnalyzer.cs

```diff
@@ -63,6 +63,11 @@ public MultiplicationOrderAnalyzer(UnityApi unityApi)
                         break;
                     }
                 }
+                
+                // incomplete expression
+                if (scalar == null)
+                    return;
+                
```

Comment:
This doesn't match the PR title. If this is fixing a YT issue, can it be added to `CHANGELOG.md`, `plugin.xml` and `resharper-unity.nuspec`?